extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -412,6 +424,33 @@ void beanDefiningAnnotations(BuildProducer<BeanDefiningAnnotationBuildItem> bean
                         BuiltinScope.SINGLETON.getName()));
     }
 
+    @BuildStep
+    void enableMetrics(ResteasyConfig buildConfig,
```

Comment:
Shouldn't this code be in the `smallrye-metrics` extension?

Reply:
All other extensions that enable metrics do it in their own processor. 
This extension has all the needed dependencies for it, Metrics extension doesn't. And it sounds odd, if it is at all possible, that Metrics extension would inject configuration of a different extension.

Reply:
I thought that those classes would be available since you added `quarkus-resteasy` as an optional dependency ni `extensions/smallrye-metrics/runtime/pom.xml`, but I'll let someone more experienced review that ;)

Reply:
That's just the runtime artifact and it's because `QuarkusJaxRsMetricsFilter` uses JAX-RS api to compile